pages/hygieia/collectors/scm/bitbucket.md

```diff
@@ -87,12 +87,18 @@ The sample `application.properties` file lists parameters with sample values to
 
 		# Mandatory parameters (comma separated, if multiple)
 		git.host=https://mybitbucketrepo.com,https://mybitbucketrepo2.com
+
+        #If using Bitbucket Cloud then go for below parameter for git.host
```

Comment:
Minor Change: Can you provide proper space before the `#If` ? Make it aligned with the `git.host`